app/javascript/admin/datatable_book.js

```diff
@@ -0,0 +1,80 @@
+import $ from 'jquery';
+global.$ = jQuery;
+
+$(document).ready(function() {
+
+	'use strict';
+
+	var datatableInit = function() {
+		var $table = $('#datatable-book');
+
+		// format function for row details
+		var fnFormatDetails = function( datatable, tr ) {
+			var data = datatable.fnGetData( tr );
+			return tr.querySelector("#table_hidden_data").innerHTML;
+		};
+
+		// insert the expand/collapse column
+		var th = document.createElement( 'th' );
+		var td = document.createElement( 'td' );
+		td.innerHTML = '<i data-toggle class="fa fa-plus-square-o text-primary h5 m-none" style="cursor: pointer;"></i>';
+		td.className = "text-center";
+
+		$table
```

Comment:
Can you fix indentation in this file? I think you need to set your ide to autoconvert tab to spaces.

Reply:
done